bcrs-ui/src/plugins/vuetify.js

```diff
@@ -0,0 +1,7 @@
+import Vue from 'vue'
```

Comment:
This is odd: I see another file with the same name and content below !?!?

Reply:
Very odd! I wonder how the .js file was created. I removed it.